### ISSUE TITLE
"fr.pilato.elasticsearch.crawler.fs.FsCrawlerImpl.getFsParser()" is null

### ISSUE THREAD

**Issue by yuanzhian** [dadoonet/fscrawler#1081]
**Describe the bug**

 Fatal error received while running the crawler: [Cannot invoke "fr.pilato.elasticsearch.crawler.fs.FsParser.isClosed()" because the return value of "fr.pilato.elasticsearch.crawler.fs.FsCrawlerImpl.getFsParser()" is null]

**Job Settings**

```yml
---
name: "test"
fs:
  url: "/tmp/es"
  update_rate: "15m"
  excludes:
  - "*/~*"
  json_support: false
  filename_as_id: false
  add_filesize: true
  remove_deleted: true
  add_as_inner_object: false
  store_source: false
  index_content: true
  attributes_support: false
  raw_metadata: false
  xml_support: false
  index_folders: true
  lang_detect: false
  continue_on_error: false
  ocr:
    language: "eng"
    enabled: true
    pdf_strategy: "ocr_and_text"
  follow_symlinks: false
elasticsearch:
  nodes:
  - url: "http://127.0.0.1:9200"
  bulk_size: 100
  flush_interval: "5s"
  byte_size: "10mb"
  ssl_verification: true

```

**Logs**

```
15:01:54,560 [32mINFO [m [f.p.e.c.f.c.BootstrapChecks] Memory [Free/Total=Percent]: HEAP [235.1mb/4gb=5.74%], RAM [132.3mb/16gb=0.81%], Swap [1.1gb/13gb=8.68%].
15:01:55,271 [32mINFO [m [f.p.e.c.f.c.v.ElasticsearchClientV7] Elasticsearch Client for version 7.x connected to a node running version 7.10.2
15:01:55,412 [32mINFO [m [f.p.e.c.f.c.v.ElasticsearchClientV7] Elasticsearch Client for version 7.x connected to a node running version 7.10.2
15:01:55,421 [32mINFO [m [f.p.e.c.f.FsCrawlerImpl] FS crawler [test2] stopped
15:01:55,421 [1;31mFATAL[m [f.p.e.c.f.c.FsCrawlerCli] Fatal error received while running the crawler: [Cannot invoke "fr.pilato.elasticsearch.crawler.fs.FsParser.isClosed()" because the return value of "fr.pilato.elasticsearch.crawler.fs.FsCrawlerImpl.getFsParser()" is null]
15:01:55,422 [32mINFO [m [f.p.e.c.f.FsCrawlerImpl] FS crawler [test2] stopped

```

**Expected behavior**

bin/fscrawler test

**Versions:**
 - OS: 10.15.3
 - Version: fscrawler-es7-2.7-SNAPSHOT

**Comment by dadoonet** [comment]
Which exact version did you downloaded?
I think this has been solved recently with #1066 and is a duplicate of #1065.

**Comment by yuanzhian** [comment]
https://oss.sonatype.org/content/repositories/snapshots/fr/pilato/elasticsearch/crawler/fscrawler-es7/2.7-SNAPSHOT/fscrawler-es7-2.7-20201223.063716-150.zip

**Comment by dadoonet** [comment]
The latest build is this one: https://oss.sonatype.org/content/repositories/snapshots/fr/pilato/elasticsearch/crawler/fscrawler-es7/2.7-SNAPSHOT/fscrawler-es7-2.7-20210121.084108-154.zip

Please use it instead.

**Comment by yuanzhian** [comment]
**logs**
```
2021-01-21 16:55:34,699 [[1;31mERROR[m] [432b55eaff4760a17c1e7b1732b55968][/zcy-m-front-master-dbdc3f8ef600dc07700cf477c8bc521492c54ba2/plop-templates/view/index.less] InputStream must have > 0 bytes

```

Empty file, how to skip, where to configure ?

**Comment by yuanzhian** [comment]
**logs**
```
11:43:30,700 [32mINFO [m [f.p.e.c.f.c.BootstrapChecks] Memory [Free/Total=Percent]: HEAP [235.4mb/4gb=5.75%], RAM [91mb/16gb=0.56%], Swap [1.8gb/15gb=12.37%].
11:43:31,084 [32mINFO [m [f.p.e.c.f.FsCrawlerImpl] Starting FS crawler
11:43:31,084 [32mINFO [m [f.p.e.c.f.FsCrawlerImpl] FS crawler started in watch mode. It will run unless you stop it with CTRL+C.
11:43:32,365 [32mINFO [m [f.p.e.c.f.c.v.ElasticsearchClientV7] Elasticsearch Client for version 7.x connected to a node running version 7.9.3
11:43:33,411 [32mINFO [m [f.p.e.c.f.c.v.ElasticsearchClientV7] Elasticsearch Client for version 7.x connected to a node running version 7.9.3
11:43:33,997 [32mINFO [m [f.p.e.c.f.FsParserAbstract] FS crawler started for [test2] for [/tmp/es] every [15m]

```
**results**
![38D702E2-5691-4128-B8E7-D87FE591D113.png](https://sitecdn.zcycdn.com/f2e-assets/a8ddad3c-deb5-4974-a8f1-24be047c5928.png?x-oss-process=image/quality,Q_75/format,jpg)

**Comment by dadoonet** [comment]
Could you open a new issue for this one? Could you run with `--trace` option and share the full stacktrace if any?

This should not happen.

I'm closing the current issue as the problem has been solved by #1066.
